tardis/montecarlo/packet_source.py

```diff
@@ -108,8 +111,25 @@ class BlackBodySimpleSource(BasePacketSource):
     """
 
     def create_packets(self, T, no_of_packets):
-        nus = self.create_blackbody_packet_nus(T, no_of_packets)
-        mus = self.create_zero_limb_darkening_packet_mus(no_of_packets)
+        self.T = T
+        self.seeds = mc_config_module.packet_seeds
+        nus = self.create_nus()
+        mus = self.create_mus()
         energies = self.create_uniform_packet_energies(no_of_packets)
+        return nus, mus, energies
+
+    def create_nus(self):
+        xis = np.zeros((5, len(self.seeds)))
+        for i, seed in enumerate(self.seeds):
+            xis[:, i] = self.create_array(seed)
+        return self.create_blackbody_packet_nus(self.T, xis)
+
+    @staticmethod
+    @njit
+    def create_array(seed):
+        np.random.seed(seed)
+        return np.random.random(5)
```

Comment:
why 5?

Reply:
I'm not really sure — I was matching the shape of the original `xis` shape within the blackbody function. I'll take a closer look at the reference material!

Reply:
Had to with the sampling method; I've added further documentation that should clear it up.